app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -217,8 +217,9 @@ def index_relation_content_unit(query)
     param_group :repo
     def create
       repo_params = repository_params
-      unless RepositoryTypeManager.creatable_by_user?(repo_params[:content_type])
-        msg = _("Invalid params provided - content_type must be one of %s") % RepositoryTypeManager.creatable_repository_types.keys.join(",")
+      unless RepositoryTypeManager.creatable_by_user?(repo_params[:content_type], false)
+        msg = _("Invalid params provided - content_type must be one of %s") %
+          RepositoryTypeManager.creatable_repository_types(false).keys.join(",")
```

Comment:
Shouldn't this validation catch this?  https://github.com/Katello/katello/blob/master/app/models/katello/root_repository.rb#L83-L87

Reply:
This is a little different since it's using `creatable_repository_types`.  I only changed the `creatable_repository_types` to have it use the defined types with that new `false` I'm passing in.

Reply:
ah, makes sense